src/AutoMapper/AutoMapperConfigurationException.cs

```diff
@@ -48,7 +48,7 @@ public override string Message
                         string.Format(
                             "The following property on {0} cannot be mapped: \n\t{2} \nAdd a custom mapping expression, ignore, add a custom resolver, or modify the destination type {1}.",
                             Types?.DestinationType.FullName, Types?.DestinationType.FullName,
-                            PropertyMap?.DestinationProperty.Name);
+                            PropertyMap?.DestinationMember.Name);
```

Comment:
Abstract base class I think - there's enough common behavior (especially around the methods)

Reply:
So instead of a base class I just moved some of the nonsensical things out, things that were too different. ConstructorParameterMaps for example don't have a `MemberInfo`.